src/core/dataProviders/authProvider.js

```diff
@@ -1,32 +1,42 @@
+import { get } from "lodash";
+import jwt from "jsonwebtoken";
 import { AUTH_LOGIN, AUTH_LOGOUT, AUTH_ERROR, AUTH_CHECK, AUTH_GET_PERMISSIONS } from 'react-admin';
+
 import { token, domainName } from "../token";
```

Comment:
domainName should come from "window.config"

Reply:
Done